src/applications/rated-disabilities/components/NeedHelp.jsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import { CONTACTS } from '@department-of-veterans-affairs/component-library/contacts';
+
+export default function NeedHelp() {
+  return (
+    <va-need-help class="vads-u-margin-y--3">
+      <div slot="content">
+        <p>
+          You can call us at <va-telephone contact={CONTACTS.VA_BENEFITS} />.
```

Comment:
This component looks different then what we currently have is this okay? I would also add a screen shot of the before and after

Reply:
Yeah that's intentional. I wanted to make sure we were using the web-component and most of the other places where we mention calling a VA phone number include the 711 sentence I added as well